v03_pipeline/lib/tasks/update_lookup_table_with_deleted_families.py

```diff
@@ -0,0 +1,50 @@
+import hail as hl
+import luigi
+
+from v03_pipeline.lib.misc.lookup import (
+    remove_family_guids,
+)
+from v03_pipeline.lib.tasks.base.base_lookup_table_task import BaseLookupTableTask
+from v03_pipeline.lib.tasks.update_variant_annotations_table_with_deleted_families import (
+    UpdateVariantAnnotationsTableWithDeletedFamiliesTask,
+)
+
+
+class UpdateLookupTableWithDeletedFamiliesTask(BaseLookupTableTask):
+    project_guid = luigi.Parameter()
+    family_guids = luigi.ListParameter()
+
+    def requires(self) -> luigi.Task:
+        return UpdateVariantAnnotationsTableWithDeletedFamiliesTask(
```

Comment:
why is `UpdateVariantAnnotationsTableWithDeletedFamiliesTask` a requirement of ` UpdateLookupTableWithDeletedFamiliesTask` and not vice versa? 

I ask because I noticed that the UpdateVariantAnnotationsTable calls `remove_family_guids` on the lookup table, which wouldn't need to happen if the requirements were reversed

Reply:
Yes, this is actually pretty tricky and probably needs better in-code comments.  We're using the lookup table itself to determine which rows of the annotations table require an update.  The rows where the family is defined are the rows that require re-annotation with `gt_stats`.  If we update the lookup table first, that information is lost and we'd have to re-annotate the entire annotations table (not the worst thing, but is computationally more expensive).

Reply:
ah nice a comment would be helpful